### ISSUE TITLE
Follow the correct rules for indentation of lists!

### ISSUE THREAD

**Issue by tedbouskill** [DavidAnson/markdownlint#1343]
When generating lists the following is valid

* item 1
* item 2
    * sub-item 1
    * sub-item 2
* item 3

Your markdown tool flags that as a violation. I refuse to use a product that makes up it's own rules

**Comment by DavidAnson** [comment]
No violations are reported for the standard Markdown expression of the list you show above: https://dlaa.me/markdownlint/#%25m%23%20Issue%201343%0A%0A-%20item%201%0A-%20item%202%0A%20%20-%20sub-item%201%0A%20%20-%20sub-item%202%0A-%20item%203%0A

**Comment by tedbouskill** [comment]
Try it in Visual Studio Code. I kept getting warnings until I uninstalled your extension.

**Comment by DavidAnson** [comment]
The properly-formatted Markdown nested list example I provided via the link above produces no warnings using the vscode-markdownlint extension version 0.55.0 for VS Code. If you provide an example that does produce a violation, I can say whether that is expected or not.

**Comment by DavidAnson** [comment]
It looks like I can get at the Markdown for your original post:
```md
* item 1
* item 2
    * sub-item 1
    * sub-item 2
* item 3
```

The markdownlint warnings are self-explanatory:
```
MD007 / ul-indent Unordered list indentation [Expected: 2; Actual: 4]
MD007 / ul-indent Unordered list indentation [Expected: 2; Actual: 4]
```

While 4 spaces of indent work in this case, the typical way to write that is via 2 spaces which is what the rule recommends.

**Comment by tedbouskill** [comment]
OK, making a recommendation a rule is annoying. I tried using asterisks and your extension marked them as incorrect.

![MD004 False Error](https://github.com/user-attachments/assets/29661302-26e7-49c1-9630-9ed68332a3c8)

Here is an example from the Markdown Guide site. Notice that they allow asterisks and that the indentation is four characters long!

![Markdown Rules](https://github.com/user-attachments/assets/9626844d-fdeb-4a48-936d-9d87541fcb1f)

Now I have to uninstall your extension again. I appreciate the effort but it's still not correct.

**Comment by DavidAnson** [comment]
Linters tend to be opinionated, but I acknowledge they are not for everyone. I will close this issue as the current rule behavior is intended. Thanks for your time.

**Comment by ccoVeille** [comment]
... Let's say I'm impressed by your apparent calm @DavidAnson

Didn't see such behavior for a while
